examples/demo-rollup/rollup_config.toml

```diff
@@ -1,6 +1,6 @@
 # We define the rollup's genesis to occur at Celestia block number `start_height`. The rollup will ignore
 # any Celestia blocks before this height
-start_height = 671431
+start_height = 1
```

Comment:
Is this ok for the case when we run with real celestial?

Reply:
Ah, we use a different config file for benchmarks. Changing this part (It was from before when I didn't have a different config file for bench)